pkgs/tools/security/imhex/default.nix

```diff
@@ -47,13 +46,7 @@ stdenv.mkDerivation rec {
     xlibs.libXdmcp
   ];
 
-  configurePhase = ''
-    cmake -DCMAKE_BUILD_TYPE=Release .
-  '';
-
-  buildPhase = ''
-    make -j
-  '';
+  cmakeFlags = ["-DCMAKE_BUILD_TYPE=Release"];
```

Comment:
that's the default: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L107